### PR TITLE
Migration of metal3 helm charts from rancher-sandbox

### PR DESCRIPTION
**Pull request by dbw7**
Just about wrapped up the migration.

Would love confirmation on whether or not the migration is correct or if I need to change anything. Additionally, not sure if the new index.html and new readme.md is correct either.

Lastly, I know Nicolas mentioned adding an app-readme.md but I didn't do that in this PR cause I didn't know what to put in there.

### CODE REVIEW

File: README.md
Comment (diconico07) [inline README.md:27]: Might be a dumb question, but why "Metal3-deploy" rather than just "Metal3" ?

File: packages/metal3-deploy/generated-changes/dependencies/external-dns/dependency.yaml
Comment (diconico07) [inline packages/metal3-deploy/generated-changes/dependencies/external-dns/dependency.yaml:1]: In #5 @jdob imported a tarball for this one, if this is because the chart is not maintained by us, we may want to use upstream url here and not copy it in our tree ?

**Review by diconico07 — COMMENTED** [review]
I did not test the chart is working (as it is an import I believe it is already tested), but overall LGTM, I just have some minor questions (so no change required but would like answers before approval).

About the `app-readme` the content is supposed to be a description of the chart and how to use it and/or any specificity for its configuration (see https://github.com/rancher/charts/blob/dev-v2.7/charts/epinio/102.0.2%2Bup1.6.2/app-readme.md for an example). It will be displayed in Rancher UI when clicking on the chart in App menu.

File: README.md
Comment (dbw7) [reply README.md:27]: This was one that was a 50/50 decision, I just stuck with the name of the chart here. Will change it to just Metal3. Is the description for it alright?

File: packages/metal3-deploy/generated-changes/dependencies/external-dns/dependency.yaml
Comment (dbw7) [reply packages/metal3-deploy/generated-changes/dependencies/external-dns/dependency.yaml:1]: In this case, my best guess in interpreting the developing doc + incorporating that would be to change [packages/external-dns/package.yaml](url) to go from "local" to the upstream url, while keeping url: packages/external-dns within metal3 dependencies.

I could be entirely wrong here, the reason I say this is cause I didn't see anything in the docs about using tarballs directly.

File: packages/metal3-deploy/generated-changes/overlay/app-readme.md
Comment (diconico07) [inline packages/metal3-deploy/generated-changes/overlay/app-readme.md:1]: This file should be directly added in the `packages/metal3-deploy/charts` directory as the chart is local.

**Review by diconico07 — APPROVED** [review]
LGTM

**Pull request merged**
